docfx_yaml/extension.py

```diff
@@ -749,7 +749,8 @@ def convert_module_to_package_if_needed(obj):
                 found_node = find_node_in_toc_tree(toc_yaml, parent_level)
 
                 if found_node:
-                    found_node.setdefault('items', []).append({'name': uid, 'uid': uid})
+                    found_node.pop('uid', 'No uid find')
```

Comment:
I think this should say, "No uid found"?

Reply:
I have a new test build is still running, will check whether these changes work tomorrow.